### DISCUSSION TITLE
Permissions for name field in Channel object type

### DISCUSSION THREAD

**Discussion by Blaza** [saleor/saleor#14446] [RFC]
Currently, in the Channel graphql type, the `name` field requires `AUTHENTICATED_APP` and `AUTHENTICATED_STAFF_USER` permissions. Can someone explain what is the rationale here? Why would we allow someone to view the channel slug, but not the name? I think we should remove the permission requirement here.

The relevant code is [here](https://github.com/saleor/saleor/blob/5fe5b50a8e1cccbdb7ed183b9be0d644e37e7cab/saleor/graphql/channel/types.py#L303):

```python
class Channel(ModelObjectType):
    id = graphene.GlobalID(required=True, description="The ID of the channel.")
    slug = graphene.String(
        required=True,
        description="Slug of the channel.",
    )

    name = PermissionsField(
        graphene.String,
        description="Name of the channel.",
        required=True,
        permissions=[
            AuthorizationFilters.AUTHENTICATED_APP,
            AuthorizationFilters.AUTHENTICATED_STAFF_USER,
        ],
    )
```

**Comment by timuric** [comment]
Hi @Blaza, this is by design. 
The channel name is for internal use only, for example, it might business name that is confidential. 
If you need to add extra information to display in the front-end, please use metadata instead.
